ui/app/vue.config.js

```diff
@@ -1,26 +1,26 @@
 module.exports = {
-  lintOnSave: false,
-  devServer: {
-    overlay: {
-      warnings: true,
-      errors: true,
+    lintOnSave: false,
+    devServer: {
+        overlay: {
+            warnings: true,
+            errors: true,
+        },
     },
-  },
-  css: {
-    extract: {
-      filename: "app.css"
-    },
-    loaderOptions: {
-      sass: {
-        additionalData: `
-          @import "normalize-scss";
-          @import "@/scss/reset.scss";
-          @import "@/scss/typography.scss";
-          @import "@/scss/variables.scss";
-          @import "@/scss/utilities.scss";
-          @import "@/scss/mixins.scss";
-        `
-      }
+    css: {
+        extract: {
```

Comment:
Is this reformatting required? Maybe I am wrong but it looks like prettier isn't setup correctly or wasn't setup correctly either on the project or in editor.